lua/orgmode/utils/fs.lua

```diff
@@ -111,7 +111,12 @@ function M.trim_common_root(paths)
     end, paths)
   end
 
-  local root = '/' .. table.concat(common_root, '/') .. '/'
+  local root
+  if vim.fn.has('win32') == 1 then
+    root = table.concat(common_root, '/') .. '/'
+  else
+    root = '/' .. table.concat(common_root, '/') .. '/'
+  end
```

Comment:
We can simplify it a bit. Only difference is that on non-Windows we need `/` prefix.
```suggestion
  local root = table.concat(common_root, '/') .. '/'
  if vim.fn.has('win32') == 0 then
    root = '/' .. root
  end
```